cpp/tests/test_reduction.cpp

```diff
@@ -30,74 +28,62 @@
 using namespace legate::dataframe;
 
 template <typename T>
-struct ReductionTest : public cudf::test::BaseFixture {};
+struct ReductionTest : public testing::Test {};
 
-TYPED_TEST_SUITE(ReductionTest, cudf::test::NumericTypes);
+TYPED_TEST_SUITE(ReductionTest, NumericTypes);
 
 TYPED_TEST(ReductionTest, Max)
 {
-  cudf::test::fixed_width_column_wrapper<TypeParam> col({5, 6, 7, 8, 9}, {1, 0, 1, 0, 1});
-  auto const type = static_cast<cudf::column_view>(col).type();
-  auto lg_col     = LogicalColumn{col};
+  auto agg = cudf::make_max_aggregation<cudf::reduce_aggregation>();
+  LogicalColumn col(narrow<TypeParam>({5, 6, 7, 8, 9}), {1, 0, 1, 0, 1});
 
-  auto agg       = cudf::make_max_aggregation<cudf::reduce_aggregation>();
-  auto res_dtype = cudf::detail::target_type(type, agg->kind);
+  auto res_dtype = cudf::detail::target_type(col.cudf_type(), agg->kind);
+  auto res       = reduce(col, "max", res_dtype);
 
-  auto expected   = cudf::reduce(col, *agg, res_dtype);
-  auto res        = reduce(lg_col, *agg, res_dtype);
-  auto res_scalar = res.get_cudf_scalar();
+  auto expected = ARROW_RESULT(arrow::compute::CallFunction("max", {col.get_arrow()})).scalar();
+  auto expected_array = ARROW_RESULT(arrow::MakeArrayFromScalar(*expected, 1));
 
-  CUDF_TEST_EXPECT_COLUMNS_EQUAL(cudf::make_column_from_scalar(*res_scalar, 1)->view(),
-                                 cudf::make_column_from_scalar(*expected, 1)->view());
+  EXPECT_TRUE(res.get_arrow()->ApproxEquals(expected_array))
```

Comment:
```suggestion
  EXPECT_TRUE(res.get_arrow()->Equals(expected_array))
```
I suppose it doesn't matter, but seems might as well use exact version.